### PR TITLE
Fix install issues

### PR DESCRIPTION
**Pull request by hemmelig**
## What is the purpose of this Pull Request? 
To address the issues arising from the recent pandas release (1.1), which made some changes to address issues arising from the recent matplotlib release (3.3).

Also iron out some minor issues with the installation process guide.

## Test cases
Ran the install with the version pins, as well as testing whether pandas 1.1 + matplotlib 3.3 worked (it did not)

- [x] All examples run without any new warnings
- [x] test_benchmarks.py reports all example tests pass

### System details
Ubuntu 18.04

### Final checklist
- [x] `develop` base branch selected?
- [x] All tests still pass.

### CODE REVIEW

**Comment by TomWinder** [comment]
Is this still the best approach to take here re. version requirements - i.e. have subsequent matplotlib / pandas updates solved this problem externally?

**Review by TomWinder — APPROVED** [review]
A few minor comments, but all works as expected (tested on Ubuntu 18.04 with a fresh conda environment installed from the updated .yml file).

Also confirmed that the current latest versions of matplotlib and pandas still do not play ball with each other. This check was done before removing the last few instances of using pandas Series directly, though (i.e. changing to .values).

File: docs/installation.rst [hunk 1]
```diff
@@ -29,17 +30,28 @@ This will install the explicit dependencies of QuakeMigrate (as well as some add
...
 #########
-We recommend you follow the installation instructions available on the website. Once you have compiled the source code, we recommend you add the bin to your system path. For Unix systems, this can be done by adding the following to your .bashrc file (typically found in your home directory, ``~/``):
+To download, unpack, and compile NonLinLoc, you can use:
+
+.. code-block:: bash
+    
+    curl http://alomax.free.fr/nlloc/soft7.00/tar/NLL7.00_src.tgz -o NLL7.00_src.tgz
+    tar -xzvf NLL7.00_src.tgz
+    cd src
+    make -R all
```
Comment (hemmelig) [inline docs/installation.rst:52]:
```suggestion
make -R all
mkdir bin; export MYBIN=./bin
```

File: docs/installation.rst [hunk 2]
```diff
@@ -29,17 +30,28 @@ This will install the explicit dependencies of QuakeMigrate (as well as some add
...
 - numpy
 - obspy >= 1.2
-- pandas >= 1
+- pandas >= 1 and < 1.1
 - pyproj >= 2.6
 - scipy
 
 .. note:: These version pins are subject to change. We defer to ObsPy to select suitable versions for NumPy/SciPy.
 
+.. warning:: Some changes to datetime handling were introduced in matplotlib 3.3, which caused some conflicts with pandas versions <= 1.0.5. It is best to
```
[the remark this answers is not in the document]
Comment (hemmelig) [reply docs/installation.rst:39]:
```suggestion
.. warning:: Some changes to datetime handling were introduced in matplotlib 3.3, which caused some conflicts with pandas versions <= 1.0.5. We have pinned these two packages for the time being until we find time to resolve the issue.
```

File: docs/installation.rst [hunk 3]
```diff
@@ -29,17 +30,29 @@ This will install the explicit dependencies of QuakeMigrate (as well as some add
...
-We recommend you follow the installation instructions available on the website. Once you have compiled the source code, we recommend you add the bin to your system path. For Unix systems, this can be done by adding the following to your .bashrc file (typically found in your home directory, ``~/``):
+To download, unpack, and compile NonLinLoc, you can use:
+
+.. code-block:: bash
+    
+    curl http://alomax.free.fr/nlloc/soft7.00/tar/NLL7.00_src.tgz -o NLL7.00_src.tgz
+    tar -xzvf NLL7.00_src.tgz
+    cd src
+    make -R all
+    mkdir bin; export MYBIN=./bin
```
Comment (hemmelig) [inline docs/installation.rst:53]:
```suggestion
    mkdir bin; export MYBIN=./bin
    make -R all
```

**Pull request merged**